src/main/java/rape/brutal/gitstarship/ammunition/Magazine.java

```diff
@@ -6,17 +6,26 @@
 package rape.brutal.gitstarship.ammunition;
 
 import rape.brutal.gitstarship.IValuable;
+import rape.brutal.gitstarship.ammunition.bullet.UsualBullet;
 
 /**
  * Created by haze on 22.03.2017.
  */
 public class Magazine implements IValuable {
 
-    private int capacity;
-    private Ammo[] ammo;
-    private String name;
-    private int size;
-    private int cost;
+    protected int capacity;
+    protected Ammo[] ammo;
+    protected String name;
+    protected int size;
+    protected int cost;
+
+    public Magazine(int capacity, String name, int size, int cost) {
```

Comment:
You set capacity here, but don't use it with array initialization.

Array always have 30 bullets, but you can have different capacity.